### ISSUE TITLE
CBEAM default on field 8

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SteveDoyle2/pyNastran#37]
```
CBEAM field 8 lists 2 defaults
early in the QRG says bit=0.0 is the default, later it says offt='GGG' is...
```

Original issue reported on code.google.com by `mesheb82` on 3 Nov 2011 at 7:00

**Comment by SteveDoyle2** [comment]
Also NX Nastran requires a blank OFFT value.  What does this correspond to in MSC Nastran?

**Comment by SteveDoyle2** [comment]
I think it's finally done.  The default is GGG because MSC/NX nastran uses that as the default.  However, because it has to be different, NX requires that that field is blank.   Thus, the field will be written as blank.

I'll call this as fixed in v0.8 because it last changed sometime before then.
